the-collab-lab/tcl-17-smart-shopping-list
# Link Add Item view at the top of List view

## Description


Add Item link with icon added to List component and placed above the search bar. This link makes it easier for the user to navigate to the Add Item view. If they have a long list they wont need to scroll to the bottom of the page to add an item.
Originally styled as a button but from an accessibility standpoint a link would be better. 

## Related Issue


closes #49

## Acceptance Criteria

- [x] Add Item ~button~ link and icon appear above the filter items search box
- [x] Add Item ~button~ link at top of page successfully reroutes user to the add view

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![Screen Shot 2021-02-24 at 9 56 31 PM](https://user-images.githubusercontent.com/47228065/109096245-3431c080-76eb-11eb-8fb6-7bb77a10d287.png)


### After

![add-item-top-revised](https://user-images.githubusercontent.com/47228065/109218729-c934c800-7784-11eb-96a4-e5ca3c8b9621.gif)


## Testing Steps / QA Criteria


From your terminal, pull down this branch with `git pull origin DD-add-item-button-top` and check that branch out with `git checkout DD-add-item-button-top`
Then type `npm start` in your terminal to launch the app.

OR 
Click on the details button below to open the Netlify preview
<img width="660" alt="Screen Shot 2021-02-23 at 9 08 15 PM" src="https://user-images.githubusercontent.com/47228065/109096599-d782d580-76eb-11eb-912a-39ff7df4ac88.png">

- Any token will do (I used `mean evict pappy`)
- Navigate to the `List` view
- Click on the `Add Item` link at the top of the page to be routed to the `Add Item` View
